src/store/selectors/index.js

```diff
@@ -1,36 +1,42 @@
 import { createSelector } from 'reselect'
-import { mapValues, every } from '../../utils/func.js'
-import { createSelectorWhichMapsToHash } from './utils.js'
-
-const captures = state => state.captures
-const createSelectorWhichMapsToCaptures = (mapFunc) =>
-        createSelectorWhichMapsToHash(captures, mapFunc)
-
-export const isThereAValidCapture = createSelectorWhichMapsToCaptures(capturesOfAType =>
-  capturesOfAType.some(i => i.valid))
-
-export const isThereAValidAndConfirmedCapture = createSelectorWhichMapsToCaptures(capturesOfAType =>
-  capturesOfAType.some(i => i.valid && i.confirmed))
-
-export const validCaptures = createSelectorWhichMapsToCaptures(capturesOfAType =>
-  capturesOfAType.filter(i => i.valid))
-
-export const unprocessedCaptures = createSelectorWhichMapsToCaptures(capturesOfAType =>
-    capturesOfAType.filter(i => !i.processed))
+import mapValues from 'object-loops/map'
+
+const currentCaptures = (state, props) => {
```

Comment:
I would write `props` as `{method, side = null}`, quicker to understand what the functions does that way